### PR TITLE
stop server alternate

### PR DESCRIPTION
**Pull request by FarajiA**
## JIRA:
N/A

## Description:
Stop server alternate

## Testing:
`npm run reference` or `npm run visual-test`

## Screenshots:
(if applicable)

### CODE REVIEW

File: backstop.js [hunk 1]
```diff
@@ -2,8 +2,11 @@ const glob = require('glob-promise');
 const backstop = require('backstopjs');
 var args = require('minimist')(process.argv.slice(2));
 const componentPath = './src/components';
-const killServer = require('./bin/stop');
+//const killServer = require('./bin/stop');
```
Comment (kyleondata) [inline backstop.js]: leftovers?

File: bin/server.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+'use strict';
+const CONFIG = require('../_config');
+const express = require('express');
+
+// Create Express App
+const app = express();
+app.use('/hedwig', express.static(CONFIG.srcComponents));
+app.use('/', express.static(CONFIG.srcComponents));
```
Comment (kyleondata) [inline bin/server.js]: not really asking for a change here, more of just a question.  What is the difference between `/hedwig` and `/` ?  It appears that they are both serving up the same path.

File: backstop.js (hunk 1 above)
Comment (FarajiA) [reply backstop.js]: Yup. Doing a cleanup now

**Comment by FarajiA** [comment]
Should be good now

**Pull request merged**